CONTRIBUTING.md

```diff
@@ -302,5 +302,7 @@ Following the targets as an example:
 Currently, kURL does not execute the checks after it be installed. However, you might want try out
 run the checks by `kubectl get -oyaml "$(kubectl get installer -oname)" | sudo kurl/bin/kurl host preflight -`. 
 
+### Why can we not run the scripts under `$HOME/kurl` and the [bin/watchrsync.js](https://github.com/replicatedhq/kURL/blob/799db33f66f91b0680facf7c14e1222798021c57/bin/watchrsync.js#L29-L32) has moved the scripts which can be executed to the `$HOME` directory?
 
+Currently, kURL scripts does not work with relative paths. For developemnt enviorment it execute operations in many places using the directory prefixed with `"$DIR". "$DIR" == `./kurl`` and for production it will be `/var/lib/kurl`. ([example](https://github.com/replicatedhq/kURL/blob/aea79861716d66787f0b31670f1fc74a7ee16d1f/scripts/common/rook.sh#L202))  
```

Comment:
```suggestion
Currently, the kURL scripts does not work with relative paths. The kURL script execute operations in many places using the directory prefixed with `"$DIR"`. In the development environment `"$DIR" == ./kurl` and for production it will be `/var/lib/kurl`. ([example](https://github.com/replicatedhq/kURL/blob/aea79861716d66787f0b31670f1fc74a7ee16d1f/scripts/common/rook.sh#L202))  
```